src/utils/MultiParText.h

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------*\
+                          ____  _ _ __ _ __  ___ _ _
+                         |_ / || | '_ \ '_ \/ -_) '_|
+                         /__|\_, | .__/ .__/\___|_|
+                             |__/|_|  |_|
+\*---------------------------------------------------------------------------*/
+#ifndef UTILS_MULTIPARTEXT_H
+#define UTILS_MULTIPARTEXT_H
+
+#include <string>
+#include <initializer_list>
+
+#include <zypp/base/String.h>	// zypp::asString
+
+///////////////////////////////////////////////////////////////////
+/// \brief Helper to build multi paragraph description texts
+///
+/// We prefer translated description paragraphs to contain no
+/// format information (parindent,parsep,NL). The class joins
+/// multiple paragraphs passed to the ctor so they are rendered
+/// correctly.
+///
+struct MultiParText
+{
+  MultiParText() = default;
+
+  template <typename Tstr>
+  MultiParText( Tstr str_r )
+  : _text { zypp::asString(str_r) }
+  {}
+
+  template <typename... Targs >
+  MultiParText( Targs... args_r )
+  { int __attribute__((unused)) dummy[] = { 0, ( (void)appender( _text, std::forward<Targs>(args_r) ), 0 )... }; }
```

Comment:
C++ is so beautful sometimes :D

Reply:
Yes. The initial version used recursive ctor calls, but thats R2L and you have to prepend each par to _text. This one is at least L2R.